frontend/packages/knative-plugin/src/components/overview/__tests__/RevisionsOverviewList.spec.tsx

```diff
@@ -28,13 +32,13 @@ describe('RevisionsOverviewList', () => {
     ).toEqual('Revisions');
   });
 
-  it('should show info if no Revisions present and traffic split sshould button should be disabled', () => {
+  it('should show info if no Revisions present, link for revisions should not be shown and traffic split button should be disabled', () => {
     const spyUseAccessReview = jest.spyOn(utils, 'useAccessReview');
     spyUseAccessReview.mockReturnValue(true);
     wrapper = shallow(
       <RevisionsOverviewList revisions={[]} service={MockKnativeResources.revisions.data[0]} />,
     );
-    expect(wrapper.find('span')).toHaveLength(1);
+    expect(wrapper.find(utils.ResourceLink)).toHaveLength(0);
```

Comment:
check for message block as well i.e in case of no revisions

Reply:
there is already a check in place for the case where there are no revisions